### DISCUSSION TITLE
Markdownpedia Back-end Changes

### DISCUSSION THREAD

**Discussion by mamamia5x** [mamamia5x/markdownpedia#28] [News]
Hello everyone, thanks for spending your time and reading this. I've come across an issue about the site, and it's something that was going to be addressed. The site itself will stay the same, but we're going to change some things up in the back-end side of the site.

What we want for Markdownpedia is to have the ability to edit pages within the site, so anyone could edit the page. The site will need some big changes, and probably a database (but that's probably way into the future). Currently, the site could use some changes to the back-end. What does that mean? Well, the site itself will stay the same. The site will look, smell, feel, taste, and sound the same. 

My plan is to probably implement it into react-js, because it'll definitely cut back on storage. It may take a little while for the back-end to get completely set up. I won't make any changes right away, I have to view my options and pick the best selection.

If you guys have any ideas, please feel free to share it.
Thanks, 
      Bugs Bunny

**Comment by Riguidix** [comment]
I think it sounds great, it's an excellent point of view to escalate this kind of project.

We can use the MEVN or MERN stack to approach it. Think about SSR or CSR, maybe open an organization so we can have multiple repos to work with:

- Client
- Backend
    - API's
- Docs

**Reply by mamamia5x** [reply]
I'll look into it, thanks ;)
